openlibrary/admin/numbers.py

```diff
@@ -132,10 +131,9 @@ def admin_range__visitors(**kargs):
         sqlitefile = tempfile.mktemp(prefix="sqlite-")
         url = "http://www.archive.org/download/stats/numUniqueIPsOL.sqlite"
         logging.debug("  Downloading '%s'", url)
-        sqlite_contents = urllib.request.urlopen(url).read()
-        f = open(sqlitefile, "w")
-        f.write(sqlite_contents)
-        f.close()
+        sqlite_contents = requests.get(url).content
+        with open(sqlitefile, "w") as f:
```

Comment:
```suggestion
        with open(sqlitefile, "wb") as f:
```
This matches the change made at https://github.com/internetarchive/openlibrary/pull/3710/files#diff-59145b1744ce8b2c433e4df964b64fc7R134